client/components/app.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReviewList from './reviewList.jsx';
+import PageSelector from './pageSelector.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allPreviews: props.reviews,
```

Comment:
It looks like your code works. I thought that for class components you would have to use "this.props.reviews" and for functional components use "props.reviews".

Reply:
I believe that's only when you're passing props into the render